examples/electron.ipynb

```diff
@@ -51,7 +51,9 @@
     "# Just sum the events for now\n",
     "for frame in frame_events:\n",
     "    for pos in frame:\n",
-    "        image[pos[0]][pos[1]] += 1\n"
+    "        row = pos // 576\n",
```

Comment:
Can we figure out `576` here programmatically?

Reply:
Not real, its just the physical size of the detector.

Reply:
Okay. Yeah, I kind of wish that we could. But it's okay.